server/apiHandlers.go

```diff
@@ -17,13 +17,25 @@ package main
 // This is what implements the per-request logic for each API method.
 
 import (
+	"github.com/golang/glog"
+
 	"google.golang.org/genproto/googleapis/rpc/code"
 	"google.golang.org/genproto/googleapis/rpc/status"
 
 	"istio.io/mixer/adapters"
 	"istio.io/mixer/api/v1"
 )
 
+const (
+	// ServiceName is a well-known name of a fact that is passed by the caller to specify
+	// the particular config that should be used when handling a request.
+	ServiceName = "serviceName"
+
+	// PeerID is a well-known name of a fact that is passed by the caller to specify
+	// the id of the client.
+	PeerID = "peerId"
+)
+
```

Comment:
I had been thinking about these to facts as constants in the mixer, just something we assume. But in reality, we could make this configurable at the global level in the mixer. Is that a useful feature for the operator?